manifests/deployment.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: kepler-model-server-pv
+  labels:
+    type: local
+    app.kubernetes.io/component: kepler-model-server-pv
+    app.kubernetes.io/name: kepler-model-server-pv
+spec:
+  storageClassName: default
+  capacity:
+    storage: 5Gi
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: "/mnt/data"
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: kepler-model-server-pvc
+  namespace: monitoring
+spec:
+  storageClassName: default
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kepler-model-server
+  namespace: monitoring
+  labels:
+    app.kubernetes.io/component: model-server
+    app.kubernetes.io/name: kepler-model-server
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: model-server
+      app.kubernetes.io/name: kepler-model-server
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: model-server
+        app.kubernetes.io/name: kepler-model-server
+    spec:
+      volumes:
+      - name: models
+        persistentVolumeClaim:
+          claimName: kepler-model-server-pvc
+      containers:
+      - name: kepler-model-server
+        image: quay.io/sustainable_computing_io/kepler_model_server_base:latest
+        ports:
+        - containerPort: 8100
+        volumeMounts:
+          - name: models
+            mountPath: /models
+      - name: kepler-model-trainer
+        image: quay.io/sustainable_computing_io/kepler_model_server_base:latest
+        command: ["python",  "energy_scheduler.py'"]
```

Comment:
We need to add `python` to the base image `$PATH`. As of now I see this error
```
  Warning  Failed   2m34s                  kubelet  Error: container create failed: time="2022-07-18T11:54:40-04:00" level=error msg="runc create failed: unable to start container process: exec: \"python\": executable file not found in $PATH"
```

Reply:
it is installed, see [this](https://github.com/sustainable-computing-io/kepler-model-server/blob/main/server/Dockerfile#L4)